inst/EpiModel 2.0 Introduction.rmd

```diff
@@ -0,0 +1,254 @@
+
+
+# EpiModel Additions & New Features
+
+Along with the changes to EpiModel detailed in the previous section, several additions have been made to EpiModel and the EpiModel workflow to improve usability and a clearer understanding of the overall epidemic process simulated by EpiModel.
+
+## TergmLite
+
+The package ```tergmLite``` has been added as a dependent package for EpiModel. TergmLite is a stripped down version of the ```network``` package ```tergm``` and decreases computation time of models in which the network is resimulated (as in models with vital dynamics) at the expense of retained data. Details of ```tergmLite``` as well as examples of what types of research projects it may be useful for can be found through querying the help package
+
+```{r echo = TRUE, eval = FALSE}
+?tergmLite::tergmLite-package
+```
+
+The use of ```tergmLite``` is fairly straightforward: the input parameter ```tergmLite``` is passed into a ```control.net``` object as a logical indicating whether to use the full network object (```tergmLite == FALSE```) or a pared down version of the network (```tergmLite == TRUE```). As an example:
+
+```{r echo = TRUE, eval = FALSE}
+nw <- network_initialize(100)
+formation <- ~edges
+target.stats <- 30
+coef.diss <- dissolution_coefs(dissolution = ~ offset(edges), duration = 25)
+est <- netest(nw, formation, target.stats, coef.diss)
+
+init <- init.net(i.num = 5)
+param <- param.net(inf.prob = 0.1)
+
+## tergmLite = FALSE: use full network object
+control <- control.net(type = "SI", nsteps = 100, nsims = 1, tergmLite = FALSE)
+sim1 <- netsim(est, param, init, control)
+
+## tergmLite = TRUE: use reduced cross-sectional edgelist
+control <- control.net(type = "SI", nsteps = 100, nsims = 1, tergmLite = TRUE)
+sim2 <- netsim(est, param, init, control)
+```
+
+## Network Update
+
+Prior to EpiModel 2.0, the base functions of EpiModel were built using functions both from ```EpiModel``` and ```network``` packages. This intermingling of functions made it difficult for the end user to tease out where network functionality ended and EpiModel began, hampering understanding of the EpiModel workflow and the ability to build module extensions. In order to make the distinction between the two packages explicit and to make the purely EpiModel workflow clearer, a new base function, ```nwupdate.net``` has been introduced. This function pulls all interactions with the network object from base functions into ```nwupdate.net``` and is run at the end of the simulation. By segmenting the EpiModel workflow in this way, base EpiModel modules are now easier to navigate and make clear the output needed for module extensions.
```

Comment:
Make it clear that this is a module that the user does not need to update in most applications.